src/config.json

```diff
@@ -9,6 +9,7 @@
     "tagline": "containers.loginHome.tagline",
     "LocalVersionIndex": "34",
     "helpUrl": "https://github.com/Cryptonomic/T2",
+    "aboutUrl": "https://cryptonomic.tech/galleon.html",
```

Comment:
"T2" builds aren't "Galleon" builds.

Reply:
@dariuszsobkowicz This should be read from a config file.